setup.cfg

```diff
@@ -25,6 +24,7 @@ classifiers =
 packages = find:
 requires = numpy
 zip_safe = False
+setup_requires = setuptools_scm
```

Comment:
> if you don't plan on running setup.py commands directly, but also no harm in leaving it there in case you do want to do e.g. python setup.py --version for instance.

I think that having the possibility to run `python setup.py --version` is very handy for developers,
so i would keep the `setup_requires` entry in in `setup.cfg` (no need to add also numpy and jinja IMHO).